releasenotes/notes/eds-cache.yaml

```diff
@@ -4,4 +4,4 @@ area: networking
 releaseNotes:
 - |
   **Added** experimental caching for endpoints, to support large scale clusters. This feature is disabled by default and can
```

Comment:
This is enabled by default now. Just for getting it tested in CI or will we ship with it turned on? Should this note be updated.